webpack/webpack.dev.babel.js

```diff
@@ -87,8 +87,7 @@ module.exports = Object.assign({}, webpackConfig, {
     new StyleLintPlugin({
       configFile: '.stylelintrc',
       files: '**/*.less',
-      syntax: 'less',
-      failOnError: false
```

Comment:
I don't think so, because it's annoying for the build to fail just because of a lint error. This is how `eslint` behaves too; you're alerted about the warning, but the build still compiles.